pkg/kafka/topic/validators.go

```diff
@@ -32,13 +32,17 @@ func ValidateName(val interface{}) error {
 		return fmt.Errorf("topic name cannot exceed %v characters", maxNameLength)
 	}
 
+	if (name == ".") || (name == "..") {
+		return fmt.Errorf(`invalid topic name "%v"; can't be "." or ".."`, name)
```

Comment:
No need to repeat the topic name is we know it is either "." or ".." already.

```suggestion
		return errors.New(`topic name can not be "." or ".."`)
```